ansible/roles/configure/tasks/config_decap.yml

```diff
@@ -0,0 +1,31 @@
+- fail: msg="information about testbed missing."
+  when: (lo_ip is not defined) or
+        (dscp_mode is not defined) 
+        
```

Comment:
Let me ask a silly question: after the interface is configured as decap rule, is there any way to reset it back to a normal interface? If so how could we do it?